git-some.sh

```diff
@@ -18,6 +18,12 @@ if grep --quiet --invert-match '^[[:digit:]]*$' <<< $commit_count; then
   exit 1
 fi
 
+COUNT=$(/bin/find . -maxdepth 1 -name "file-??.txt" | wc --lines)
+if [[ $(($COUNT + $commit_count)) -gt 100 ]]; then
+  echo "Can't create that many files!" >&2
```

Comment:
Align with https://github.com/GROSSWEBER/git-some/blob/master/git-some.sh#L17

(no `"`, redirection in front)